content/blog/2020/08/2020-08-04-cdf-graduation.adoc

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "Jenkins graduates in the Continuous Delivery Foundation"
+tags:
+- announcement
+- cdf
+- general
+- community
+author: oleg_nenashev
+description: >
+  The Jenkins project has graduated in the Continuous Delivery Foundation (CDF).
+  In this article, we discuss what it means to the Jenkins community and what are the future steps for the project.
+opengraph:
+  image: TODO
+---
+
+On behalf of the Jenkins Governance Board,
+we are happy to announce that the Jenkins project has achieved the graduated status in
+the link:https://cd.foundation/[Continuous Delivery Foundation (CDF)].
+This status is official effective Aug 03, 2020.
+
+In this article, we will discuss what the CDF membership and graduation mean to the Jenkins community and what are the future steps for the project.
+If you are interested to know more,
+see the link:TODO[announcement on the CDF website]!
+
+== How does CDF membership help us?
+
+About 18 months ago, Jenkins link:/blog/2019/03/12/cdf-launch/[became one of the CDF founding projects],along with Jenkins X, Spinnaker and Tekton.
+A new foundation was formed to provide a vendor-neutral home for open source projects used for Continuous Delivery and Continuous Integration.
+Special interest groups were started to foster collaboration between projects and end user companies,
+most notably 
+link:https://github.com/cdfoundation/sig-interoperability[Interoperability],
+link:https://github.com/cdfoundation/sig-mlops[MLOps] and
+link:https://github.com/cdfoundation/sig-security[Security] SIGs.
+Also, a link:https://cd.foundation/ambassador-program-overview-application/[Community Ambassador] role was created to organize local meetups and to provide public-facing community representatives.
```

Comment:
```suggestion
Also, a link:https://cd.foundation/ambassador-program-overview-application/[Community Ambassador] role was created to organize local meetups and to provide public-facing community representatives.
Many former Jenkins Ambassadors and other contributors became CDF Ambassadors.
```

Reply:
I like that phrasing and the reminder that I need to apply to become a CDF Ambassador.